contrib/certbot/template.py

```diff
@@ -25,11 +42,77 @@
 maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
 license = "Apache-2.0 AND MIT"
 url = "https://github.com/certbot/certbot"
-source = (
-    f"https://github.com/certbot/certbot/archive/refs/tags/v{pkgver}.tar.gz"
-)
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
 sha256 = "7e277bb461cae4071e22641e076d9232ae00ffda05bdb02832cbc1f862afab2d"
 
 
+def post_build(self):
+    for plugin in _plugins:
+        self.do(
+            "python",
+            "-m",
+            "build",
+            "--wheel",
+            "--no-isolation",
+            f"../{plugin}",
+        )
+
+
 def post_install(self):
+    from cbuild.util import python
+
+    for plugin in _plugins:
+        self.do(
+            "python",
+            "-m",
+            "installer",
+            "--destdir",
```

Comment:
add --compile-bytecode, drop the manual precompile

Reply:
ok